tests/test_image_structure.py

```diff
@@ -0,0 +1,74 @@
+import rasterio
+
+from rasterio.enums import Compression, Interleaving
+
+
+def test_enum_compression_JPEG():
```

Comment:
@sgillies unsure of what specifically you are testing with these variants - that your enum works?  Or that names are lowercase versions of values?

Some alternatives that are more succinct:

Test for all names are lowercase versions of their values (though this just seems like a sanity check for the code):
`assert min([x.name.upper() == x.value for x in Compression]) == True`

Test that all known values are present:
`assert not set(['CCITTFAX3', 'CCITTFAX4', 'CCITTRLE', 'DEFLATE', 'JPEG', 'LZMA', 'LZW', 'NONE', 'PACKBITS']).difference([x.value for x in Compression])`